src/pieces/commandManager.ts

```diff
@@ -88,7 +88,8 @@ async function runCommand(msg: Message) {
 
 	if (msg.channel.type === 'DM' && command.runInDM === false) return msg.reply(`${command.name} is not available in DMs.`);
 	if (msg.channel.type === 'GUILD_TEXT' && command.runInGuild === false) {
-		await msg.reply('That command is not available here. Try again in DMs.');
+		await msg.author.send('The command you just tried to run is not available in public channels. Try again in DMs')
```

Comment:
Do you want to add a ping here to emulate v2.2.x?

Reply:
I don't think I know what you mean by that

Reply:
Something like this.
```suggestion
		await msg.author.send(`<@!${msg.author.id}>, The command you just tried to run is not available in public channels. Try again in DMs.`)
```